changelogs/unreleased/48112.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
Please remove this changelog entry since this is a refactor with no functional change